### ISSUE TITLE
Mentor-mentee: popup for copying Plan into Job showed when Mentor complete plan in Mentees Way

### ISSUE THREAD

**Issue by pp2amd6** [tritonJS826/masters-way#734]
Popup for copying Plan into Job showed when Mentor complete plan in Mentees Way.

Popup for copying plan into job **should not** be shown because Mentor can`t do anything in Jobs section in Mentees Way.
In that case Plan should be marked as Complete without popup showing

## Preconditions
1. The user is already authorized in the application using his Google account
2. The user has a mentee
3. Mentee way page already opened
4. Day report already created
5. Plan already created

## Steps
1. Click on checkbox "Mark the plan as complete"
![image](https://github.com/tritonJS826/masters-way/assets/4976837/1b2560f1-ca0b-43f5-8adf-f4e5cb016855)

## Actual result
Popup for copying Plan into Job showed

## Expected result
Plan should be marked as Complete without popup showing

**Comment by Ekaterina1994** [comment]
@tritonJS826  Decided allow mentors to create jobs, so it's not a bug?

**Comment by tritonJS826** [comment]
yeah, we decided to provide ability for mentors add jobs (could be useful on the first lessons to make an axample)
